src/main/scala/bubblewrap/HttpClient.scala

```diff
@@ -30,12 +27,13 @@ class HttpClient(clientSettings: ClientSettings = ClientSettings()) {
                                           .setUseInsecureTrustManager(true)
                                           .setMaxRequestRetry(clientSettings.retries)
                                           .setFollowRedirect(false)
-                                          .setKeepAlive(clientSettings.keepAlive)
-                                          .setNettyTimer(timer).build())
+                                          .setKeepAlive(clientSettings.keepAlive).setNettyTimer(timer)
+                                          .build())
 
 
   def get(url: WebUrl, config: CrawlConfig) = {
```

Comment:
Can we make `get` take an `ExecutorService` as implicit arg? More like 
```
def get(url: Weburl, config: CrawlConfig)(implicit ec: ExecutionContext)
```
This will allow us to pin a different thread-pool from the application which uses this library.

Reply:
@brewkode, Makes sense.